common/utils.js

```diff
@@ -1,30 +0,0 @@
-export function findById(cues, id) {
```

Comment:
What happened to this file? somehow it disappeared but it's on your master branch. to make it easy just copy it straight from github master branch and  paste it into your storage branch.